accounts/erd16rp9ur5crj6lcjyttr0ftft8vspmcgq3kk00wmzkv6p7lnqg3v8quhqhh5.json

```diff
@@ -1,10 +1,10 @@
 {
-  "name": "Elrond Foundation: Protocol Treasury",
+  "name": "Multiversx Foundation: Protocol Treasury",
```

Comment:
Where Multiversx starts with caps M in this file, should not be MultiversX with caps X too?

Reply:
fixed